content/kubeone/master/guides/addons/_index.en.md

```diff
@@ -53,6 +53,10 @@ data:
 
```

Comment:
I think we should get rid of the link to the design proposal on L14 because it's heavily outdated. There's an explanation about how addons work (including `--prune` and everything else) at the bottom of this document.

Reply:
👍